src/components/ha-icon-button.ts

```diff
@@ -24,28 +24,21 @@ export class HaIconButton extends LitElement {
 
   protected render(): TemplateResult {
     return html`
-      <mwc-icon-button
-        .label=${this.label}
-        ?disabled=${this.disabled}
-        @click=${this._handleClick}
-      >
+      <mwc-icon-button .label=${this.label} .disabled=${this.disabled}>
         <ha-icon .icon=${this.icon}></ha-icon>
       </mwc-icon-button>
     `;
   }
 
-  private _handleClick(ev) {
-    if (this.disabled) {
-      ev.stopPropagation();
-    }
-  }
-
   static get styles(): CSSResult {
     return css`
       :host {
         display: inline-block;
         outline: none;
       }
+      :host([disabled]) {
```

Comment:
Does mwc-icon-button reflect it? You changed it from attribute to property

Reply:
Yes